docs/sphinx/developers/wsi.rst

```diff
@@ -2,10 +2,15 @@ Working with whole slide images
 ===============================
 
 Bio-Formats supports many whole slide image formats, but effectively working with this type of data requires some extra considerations.
+Each file contains images representing one or more physical slides, with each slide typically stored at multiple resolutions.
+The width and height of a full resolution whole slide image often exceeds 100,000 pixels, so the uncompressed image size may be several GBs.
```

Comment:
exceed should be singular due to referring to plural width and height, and GBs might be better as gigabytes in prose.